public/components/content-list-drawer/content-list-drawer.js

```diff
@@ -320,6 +320,23 @@ export function wfContentListDrawer($rootScope, config, $timeout, $window, conte
                 updateField("plannedNewspaperPageNumber", pageNumber, $scope.contentItem.plannedNewspaperPageNumber)
             }
 
+            $scope.updateOffice = (office) => {
+                // preview
+                wfComposerService.updateSetting(
+                    $scope.contentItem.composerId,
+                    "productionOffice",
+                    office,
+                )
+
+                // live
```

Comment:
For fields / settings like Production Office, the draft / live distinction doesn't matter, so we update both (regardless of publication state).